cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectBuildUtils.java

```diff
@@ -83,4 +83,17 @@ public static boolean buildSucceeded(Build build)   {
     public static boolean buildComplete(Build build)   {
         return buildSucceeded(build) || buildFailed(build);
     }
+
+    /**
+     * Get ContainerStatus of the Connect build container
+     *
+     * @param pod Pod which contains ContainerStatus of the Connect build
+     *
+     * @return ContainerStatus of the build container
+     */
+    public static ContainerState getConnectBuildContainerStatus(Pod pod)   {
+        ContainerStatus containerStatus = pod.getStatus().getContainerStatuses().stream()
+            .filter(conStatus -> conStatus.getName().equals(pod.getMetadata().getName())).findFirst().orElseGet(ContainerStatus::new);
+        return containerStatus.getState() != null ? containerStatus.getState() : new ContainerState();
```

Comment:
I think this should probably return null when it is not there, or?

Reply:
In this moment we can return just `containerStatus.getState()`, as it already can contain `null`

Reply:
Yeah, sure.

Reply:
That is not exactly what I meant. There is a difference between the ContainerState being null and you creating a new ContainerState. I think you should not create it and just return the null.